pkgs/tools/misc/ufetch-nixos/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "ufetch-nixos";
+  src = ./ufetch-nixos;
```

Comment:
Use `fetchFromGitLab` on the repository or `fetchurl` on `https://gitlab.com/jschx/ufetch/-/raw/v0.2/ufetch-nixos`.